accounts/accounts_cli.go

```diff
@@ -0,0 +1,328 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/codegangsta/cli"
+	"github.com/howeyc/gopass"
+)
+
+// Represents the user running the audit on the accounts that they have access to via permissons
+type admin struct {
+	host   string
+	client *http.Client
+	User   struct {
+		Id    string `json:"userid"`
+		Name  string `json:"username"`
+		Token string `json:"-"`
+	} `json:"administrator"`
+	Accounts []*Account `json:"administeredAccounts"`
+}
+
+// Account details that we will audit
+type Account struct {
+	Id      string `json:"userid"`
+	Profile struct {
+		FullName string `json:"fullName"`
+		Patient  struct {
+			Bday string `json:"birthday"`
+			Dday string `json:"diagnosisDate"`
+		} `json:"patient"`
+	} `json:"patientProfile"`
+	Perms      interface{} `json:"permissons"`
+	LastUpload string      `json:"lastupload"`
```

Comment:
Not that we need to do anything about this now, but do we want to standardize on a capitalization standard for the any JSON?  For example, "lastupload" to "lastUpload"?  (Of course, I still don't know whether "username" is one word or two. ;)